user/src/com/google/gwt/user/tools/templates/maven/pom.xmlsrc

```diff
@@ -15,8 +15,8 @@
     <!-- Setting maven.compiler.source to something different to 1.8
          needs that you configure the sourceLevel in gwt-maven-plugin since
          GWT compiler 2.8 and above requires 1.8 (see gwt-maven-plugin block below) -->
-    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Agreed, if we're requiring java 11+, we should use release instead of both source and target. These samples won't have a difference when doing that, but others may.